prow/config.yaml

```diff
@@ -399,6 +399,30 @@ tide:
     - lgtm
     - approved
     - "cncf-cla: yes"
+    - priority/critical-urgent
+    - kind/bug
```

Comment:
Don't we just need to add these two changes
```
- priority/critical-urgent
- kind/bug 
```
or am I missing something? With this whole new section we would have slush and freeze together?